rundeckapp/grails-app/taglib/rundeck/PluginTagLib.groovy

```diff
@@ -168,4 +169,26 @@ class PluginTagLib {
             return defaultmsg
         }
     }
+
+    /**
+     * Return plugin i18n message or default without encoding it
+     *
+     *
+     * @attr json provider name
+     */
+    def customFields = { attrs, body ->
+        if(attrs.json){
+            def json = attrs.json
+            def jsonSlurper = new JsonSlurper()
+            def fields = jsonSlurper.parseText(json)
+            out << '<hr/>'
```

Comment:
better to wrap the output in a classified element, that can be styled with a border via css, rather than use `<hr/>`
```suggestion
```